transformato/testing.py

```diff
@@ -0,0 +1,111 @@
+import transformato
+import sys, os
+import logging
+import shutil
+import pathlib
+from transformato import load_config_yaml, SystemStructure, ProposeMutationRoute, IntermediateStateFactory, FreeEnergyCalculator
+import parmed as pm
+import copy
+import numpy as np
+# read in specific topology with parameters
+import subprocess
+
+# list of tuples for values nstep and nstdcd
+from numpy import arange 
+
+def createList(r1,r2,d):
+    return arange(r1,r2+1,d)
+
+
+nsteps_list = createList(10000, 100000, 5000)
+nstdcd_list = createList(1000, 10000, 1000)
+
+steps_map = [(x,y) for x in nsteps_list for y in nstdcd_list]
+
+#user_input = input("Enter the path to destination folder: ")
+#assert os.path.exists(user_input), "There is no folder "+str(user_input)
+
+from transformato import FreeEnergyCalculator
+conf = '/home/master/transformato/config/ethane-methanol-solvation-free-energy.yaml'
+configuration = load_config_yaml(config=conf,
+                                 input_dir='/home/master/transformato-systems', 
+                                 output_dir='/home/master/Master/test') #user_input 
+
+for i in steps_map:
+    configuration['simulation']['parameters']['nstep'] = i[0]
+    configuration['simulation']['parameters']['nstdcd'] = i[1]
+
+    # load systems
+    s1 = SystemStructure(configuration, 'structure1')
+    s2 = SystemStructure(configuration, 'structure2')
+    a = ProposeMutationRoute(s1, s2)
+    
+    #generate mutation_map
+    guess = [2,4,6]
+    mutation_list1 = []
+    mutation_list2 = []
+    for i in guess:
+        tmp1 = a.generate_mutations_to_common_core_for_mol1(nr_of_steps_for_el=i, nr_of_steps_for_bonded_parameters=i)
+        mutation_list1.append(tmp1)#list of lists
+        tmp2 = a.generate_mutations_to_common_core_for_mol2(nr_of_steps_for_el=i)
+        mutation_list2.append(tmp2) #list of lists
```

Comment:
this is a good way to test this approach!
But, there is one thing missing that is also hard to see and catch: `generate_mutations_to_common_core_for_mol1` always scales the `bonded_parameters` in contrast to `generate_mutations_to_common_core_for_mol2`.
Could you also include a mutation that swaps `mol1` and `mol2`?